### PR TITLE
chore: resolve inaccuracies around exposing consensus node's RPC

### LINKED ISSUE #522: Inaccuracies around exposing consensus node's RPC
In the docs under the consensus node. There is a section https://docs.celestia.org/nodes/consensus-full-node/#optional-configure-for-rpc-endpoint which I don't believe is quite accurate

> ### Optional: configure for RPC endpoint
>You can configure your Consensus Full Node to be a public RPC endpoint and listen to any connections from Data Availability > >Nodes in order to serve requests for the Data Availability API [here](https://docs.celestia.org/developers/node-tutorial/).
> 
>Note that you would need to ensure port 9090 is open for this.
>
>Run the following commands:
>
>EXTERNAL_ADDRESS=$(wget -qO- eth0.me)
>sed -i.bak -e "s/^external-address = \"\"/external-address = \"$EXTERNAL_ADDRESS:26656\"/" $HOME/.celestia-app/config/config.toml
>sed -i 's#"tcp://127.0.0.1:26657"#"tcp://127.0.0.1:26657"#g' ~/.celestia-app/config/config.toml
>
>Restart celestia-appd in the previous step to load those configs.

Populating the external address isn't necessary to making the RPC endpoint public. The `external_address` (which uses an underscore instead of a hyphen) is commonly for when you're running your node behind a NAT and so you need to advertise a different address for peers to dial. 

> sed -i 's#"tcp://127.0.0.1:26657"#"tcp://127.0.0.1:26657"#g' ~/.celestia-app/config/config.toml

This part doesn't seem to change anything. You could suggest using the public IP instead of localhost to expose the port directly or we could include documentation around setting up a reverse proxy (which I think is the standard way people expose RPC).

> Note that you would need to ensure port 9090 is open for this.

I was a bit confused about this at first because 9090 is not normally the port for RPC. As I understand it, this refers to making gRPC public as it's also required by bridge nodes. Could we better clarify this.

### PR DESCRIPTION
**Pull request by jcstein**
note: after this is implemented, i plan to complete #556

## Overview

Resolves #522 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/nodes/full-consensus-node.mdx [hunk 1]
```diff
@@ -239,21 +239,44 @@ Please refer to the [ports](../../nodes/celestia-node/#ports) section for inform
 which ports are required to be open on your machine.
 :::
 
-### Optional: configure for RPC endpoint
+### Optional: Configuration for RPC endpoint
```
Comment (jcstein) [inline docs/nodes/full-consensus-node.mdx:242]:
```suggestion
### Optional: Configuring an RPC endpoint
```

File: docs/nodes/full-consensus-node.mdx [hunk 2]
```diff
@@ -239,21 +239,44 @@ Please refer to the [ports](../../nodes/celestia-node/#ports) section for inform
...
 :::
 
-### Optional: configure for RPC endpoint
+### Optional: Configuring an RPC endpoint
 
-You can configure your full consensus node to be a public RPC endpoint
-and listen to any connections from Data Availability Nodes in order to
-serve requests for the Data Availability API [here](../developers/node-tutorial.mdx).
+You can configure your Consensus Full Node to be a public RPC endpoint.
+This allows it to accept connections from Data Availability Nodes and]
```
Comment (jcstein) [inline docs/nodes/full-consensus-node.mdx:245]:
```suggestion
This allows it to accept connections from Data Availability Nodes and
```

**Review by cmwaters — APPROVED** [review]
LGTM. Thanks for doing this

**Comment by jcstein** [comment]
you're welcome, thank you for the descriptive issue!

**Pull request merged**